teamscale-maven-plugin/src/main/java/com/teamscale/tia/maven/TiaMojoBase.java

```diff
@@ -286,19 +286,7 @@ private void validateTestPluginConfiguration(PluginExecution execution) throws M
 
 		validateParallelizationParameter(configurationDom, "threadCount");
 		validateParallelizationParameter(configurationDom, "forkCount");
-
-		Xpp3Dom parameterDom = configurationDom.getChild("reuseForks");
-		if (parameterDom == null) {
-			return;
-		}
-
-		String value = parameterDom.getValue();
-		if (value != null && !value.equals("true")) {
```

Comment:
I know this does not work with all the ways you can configure reuse forks but I think we should keep the log **as a warning** regardless. We'll need to change the wording a bit, though. We shouldn't state that it's not working at all, but has some performance impact. Maybe you can reuse parts of the docs change :)